modules/install-creating-install-config-aws-local-zones-subnets.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+[id="install-creating-install-config-aws-local-zones-subnets_{context}"]
+= Modifying an installation configuration file to use AWS Local Zones subnets
+
+Modify an `install-config.yaml` file to include AWS Local Zones subnets.
+
+.Prerequisites
+
+* You created subnets by using the procedure "Creating a subnet in AWS Local Zones".
+* You created an `install-config.yaml` file by using the procedure "Creating the installation configuration file".
+
+.Procedure
+
+* Add the VPC and Local Zone subnets as the values of the `platform.aws.subnets` property. As an example:
++
+[source,yaml]
+----
+...
+platform:
+  aws:
+    region: us-west-2
+    subnets: <1>
+    - publicSubnetId-1
+    - publicSubnetId-2
+    - publicSubnetId-3
+    - privateSubnetId-1
+    - privateSubnetId-2
+    - privateSubnetId-3
+    - publicSubnetId-LocalZone-1
+...
+----
+<1> List of subnets created in the Availability and Local Zones.
```

Comment:
It would be nice to add the `Next Steps` here pointing to `Deploying the cluster` (although it is already the next section) =)

Reply:
Makes sense to be consistent with the other installation path. 

:heavy_check_mark: